src/main/java/tc/oc/pgm/rotation/FixedPGMMapOrder.java

```diff
@@ -0,0 +1,161 @@
+package tc.oc.pgm.rotation;
+
+import com.google.common.base.Optional;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.logging.Level;
+import org.bukkit.configuration.ConfigurationSection;
+import tc.oc.pgm.api.PGM;
+import tc.oc.pgm.map.PGMMap;
+
+/** In practice, a {@link FixedPGMMapOrder} is a rotation of maps, a type of {@link PGMMapOrder} */
+public class FixedPGMMapOrder implements PGMMapOrder {
+  private ConfigurationSection configurationSection;
+
+  private String name;
+  private boolean enabled;
+  private List<PGMMap> maps = new ArrayList<>();
+  private int players;
+  private int position;
+  private boolean booting = true;
+
+  FixedPGMMapOrder(ConfigurationSection configurationSection, String name) {
+    this.configurationSection = configurationSection;
+    this.name = name;
+    this.enabled = configurationSection.getBoolean("enabled");
+    this.players = configurationSection.getInt("players");
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public boolean isEnabled() {
+    return enabled;
+  }
+
+  public void setEnabled(boolean enabled) {
+    this.enabled = enabled;
+  }
+
+  public List<PGMMap> getMaps() {
+    return Collections.unmodifiableList(maps);
+  }
+
+  public int getPlayers() {
+    return players;
+  }
+
+  public void setPlayers(int players) {
+    this.players = players;
+  }
+
+  public void setPosition(int position) {
+    this.position = position;
+  }
+
+  public int getPosition() {
+    return position;
+  }
+
+  public void load() {
+    configurationSection
+        .getStringList("maps")
+        .forEach(
+            map -> {
+              final Optional<PGMMap> mapByNameOrId =
+                  PGM.get().getMapLibrary().getMapByNameOrId(map);
+              if (mapByNameOrId.isPresent()) maps.add(mapByNameOrId.get());
+              else {
+                PGM.get()
+                    .getLogger()
+                    .warning(
+                        "[Rotation] "
+                            + "["
+                            + name
+                            + "] "
+                            + map
+                            + " not found in map repo. Ignoring...");
+              }
+            });
+
+    Optional<PGMMap> next_map =
```

Comment:
In the future, Java variables should not have `_` in them. So this would be `nextMap`. Don't need to change this now.

Reply:
Fixed https://github.com/Electroid/PGM/pull/114/commits/c6f69e3ed9c438b0e9c81ae6eaeb17762ba9463b